Makefile

```
@@ -95,7 +95,8 @@ open_xcode: build
 			$(XCODE)/Contents/MacOS/Xcode
 
 clean:
-	rm -rf /tmp/xcbuild.*
+	rm -fr /tmp/xcbuild.* && \
+	rm -fr /tmp/xcbuild-*
```

minor suggestion (took me a couple seconds to spot the difference between the lines)

```suggestion
	rm -fr /tmp/xcbuild{.,-}*
```

---

Awesome thx!